src/app/support/page.js

```diff
@@ -1,5 +1,24 @@
+import Connect from "@/components/support/Connect";
+import { SOCIALS } from "@/data/support/Socials";
+
 const Page = () => {
-  return <>Support</>;
+  return (
+    <>
+      <div className="text-center text-biscuits-brown-200 font-shrikhand p-1 -translate-y-1 text-3xl">
+        {"Connect with us!"}
```

Comment:
```suggestion
        Connect with us!
```
if its hardcoded text you can write it normally without curly braces.